linodecli/baked/operation.py

```diff
@@ -356,8 +357,12 @@ def __init__(
             self.action_aliases = {}
             self.action = action
 
-        self.summary = operation.summary
-        self.description = operation.description.split(".")[0]
+        # Ensure the summary has punctuation
+        self.summary = operation.summary.rstrip(".") + "."
+
+        self.description_rich, self.description = simplify_description(
```

Comment:
We don't consume `OpenAPIOperation(...).description` currently but I figured it'd be good to future-proof 🙂